python-cryptography build error - probably latest update openssl to v.1.0.2i

make -j8 BUILD_Log=1
[sources]/logs/package/feeds/packages/python-cryptography/compile.txt:
```
running install
running build
running build_py
running egg_info
writing requirements to src/cryptography.egg-info/requires.txt
writing src/cryptography.egg-info/PKG-INFO
writing top-level names to src/cryptography.egg-info/top_level.txt
writing dependency_links to src/cryptography.egg-info/dependency_links.txt
writing entry points to src/cryptography.egg-info/entry_points.txt
reading manifest file 'src/cryptography.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/_build'
warning: no previously-included files matching '*' found under directory 'vectors'
writing manifest file 'src/cryptography.egg-info/SOURCES.txt'
warning: build_py: byte-compiling is disabled, skipping.

running build_ext
generating cffi module 'build/temp.linux2-2.7/_padding.c'
already up-to-date
generating cffi module 'build/temp.linux2-2.7/_constant_time.c'
already up-to-date
generating cffi module 'build/temp.linux2-2.7/_openssl.c'
already up-to-date
building '_openssl' extension
powerpc-openwrt-linux-musl-gcc -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -pipe -mcpu=464fp -fno-caller-saves -fno-plt -fhonour-copts -Wno-error=unused-but-set-variable -Wno-error=unused-result -iremap /home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/cryptography-1.5:cryptography-1.5 -Wformat -Werror=format-security -fstack-protector -D_FORTIFY_SOURCE=1 -Wl,-z,now -Wl,-z,relro -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/include -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/toolchain-powerpc_464fp_gcc-5.4.0_musl-1.1.15/usr/include -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/toolchain-powerpc_464fp_gcc-5.4.0_musl-1.1.15/include/fortify -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/toolchain-powerpc_464fp_gcc-5.4.0_musl-1.1.15/include -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/python2.7 -fPIC -I/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/python2.7 -c build/temp.linux2-2.7/_openssl.c -o build/temp.linux2-2.7/build/temp.linux2-2.7/_openssl.o
build/temp.linux2-2.7/_openssl.c:1993:1: warning: function declaration isn't a prototype [-Wstrict-prototypes]
 const EC_METHOD *(*EC_GF2m_simple_method)() = NULL;
 ^
build/temp.linux2-2.7/_openssl.c:2540:14: error: conflicting types for 'COMP_METHOD'
 typedef void COMP_METHOD;
              ^
In file included from /home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/openssl/crypto.h:131:0,
                 from /home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/openssl/bio.h:69,
                 from /home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/openssl/asn1.h:65,
                 from build/temp.linux2-2.7/_openssl.c:466:
/home/sj/Pulpit/Projekty/WNDR4700-LEDE/staging_dir/target-powerpc_464fp_musl-1.1.15/usr/include/openssl/ossl_typ.h:181:31: note: previous declaration of 'COMP_METHOD' was here
 typedef struct comp_method_st COMP_METHOD;
                               ^
error: command 'powerpc-openwrt-linux-musl-gcc' failed with exit status 1
Makefile:53: recipe for target '/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/cryptography-1.5/.built' failed
make[3]: *** [/home/sj/Pulpit/Projekty/WNDR4700-LEDE/build_dir/target-powerpc_464fp_musl-1.1.15/cryptography-1.5/.built] Error 1
```

--- a/lang/python-cryptography/Makefile
+++ b/lang/python-cryptography/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cryptography
-PKG_VERSION:=1.5
+PKG_VERSION:=1.5.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://pypi.python.org/packages/6e/96/b8dab146e8be98061dae07e127f80cffa3061ab0e8da0d3d42f3308c6e91
-PKG_MD5SUM:=d85a91af3e943c68fa5ff82a3ccdd5df
+PKG_SOURCE_URL:=https://pypi.python.org/packages/21/e1/37fc14f9d77924e84ba0dcb88eb8352db914583af229287c6c965d66ba0d
+PKG_MD5SUM:=66a3e01f5f436d2413ef47b7e6bb7729
 
 PKG_BUILD_DEPENDS:=python-cffi/host
 

